pallets/ddc-clusters/src/cluster.rs

```diff
@@ -41,10 +41,7 @@ pub struct ClusterGovParams<Balance, BlockNumber> {
 	pub storage_bond_size: Balance,
 	pub storage_chill_delay: BlockNumber,
 	pub storage_unbonding_delay: BlockNumber,
-	pub unit_per_mb_stored: u128,
-	pub unit_per_mb_streamed: u128,
-	pub unit_per_put_request: u128,
-	pub unit_per_get_request: u128,
+	pub pricing: ClusterPricingParams,
```

Comment:
Can we leave the `ClusterGovParams` as a flat list and use the `ClusterPricingParams` only as a return type for the `get_pricing_params` function?

The issue here is that we are breaking compatibility with Cluster's pallet for JS SDK and apps that depend on it, and existing e2e tests. Those parts expect the `ClusterGovParams` to be a flat list currently. If we want to change the format of extrinsic params or storage item, we will have to adapt the SDK part first before proceeding with deployment to Devnet and Testnet, and redeploy apps. It will make `PayoutsPalle`t delivery longer.

One more point on why we keep the `ClusterGovParams` in extrinsic and storage as a flat list is the fact this list is not completely defined yet. We will migrate it very often, and a flat list might be easier to migrate comparing to a list of nested structs (althought should be possible)